core/src/main/java/org/lflang/generator/c/CGenerator.java

```diff
@@ -524,9 +524,11 @@ public void doGenerate(Resource resource, LFGeneratorContext context) {
             context.getMode());
         context.finish(GeneratorResult.Status.COMPILED, null);
       } else if (dockerBuild.enabled()) {
-        boolean success = buildUsingDocker();
-        if (!success) {
-          context.unsuccessfulFinish();
+        if (targetConfig.target != Target.Python) {
```

Comment:
Not clean, but a necessary hack considering the basic lack of modularity in the C/Python code generator...